<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LecteurVue</class>
 <widget class="QMainWindow" name="LecteurVue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LecteurVue</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="diapoTitle">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Nom du diaporama</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="image">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../../../../DOWNLOADS/Title.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="imageName">
          <property name="text">
           <string>Nom image</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="imageCat">
          <property name="text">
           <string>Catégorie Image</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QProgressBar" name="diapoProgress">
        <property name="value">
         <number>0</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>XX/XX</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="previousBtn">
        <property name="text">
         <string>Reculer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="modeBtn">
        <property name="text">
         <string>Démarer/Arrêter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nextBtn">
        <property name="text">
         <string>Avancer</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>838</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuDiaporama">
    <property name="title">
     <string>Paramètres</string>
    </property>
    <widget class="QMenu" name="menuMode">
     <property name="title">
      <string>Mode</string>
     </property>
     <addaction name="actionAutomatique_2"/>
     <addaction name="actionManuel_2"/>
    </widget>
    <widget class="QMenu" name="menuVitesse_de_d_filement">
     <property name="title">
      <string>Vitesse de défilement</string>
     </property>
     <addaction name="action1_seconde"/>
     <addaction name="action5_secondes"/>
     <addaction name="action10_secondes"/>
     <addaction name="actionPersonnalis"/>
    </widget>
    <addaction name="actionCharger_un_diaporama"/>
    <addaction name="actionEnlever_le_diaporama"/>
    <addaction name="separator"/>
    <addaction name="menuMode"/>
    <addaction name="menuVitesse_de_d_filement"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionA_propos"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuDiaporama"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAutomatique_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Automatique</string>
   </property>
  </action>
  <action name="actionManuel_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Manuel</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="action1_seconde">
   <property name="text">
    <string>1 seconde</string>
   </property>
  </action>
  <action name="action5_secondes">
   <property name="text">
    <string>5 secondes</string>
   </property>
  </action>
  <action name="action10_secondes">
   <property name="text">
    <string>10 secondes</string>
   </property>
  </action>
  <action name="actionPersonnalis">
   <property name="text">
    <string>Personnalisé</string>
   </property>
  </action>
  <action name="actionA_propos">
   <property name="text">
    <string>A propos</string>
   </property>
  </action>
  <action name="actionEnlever_le_diaporama">
   <property name="text">
    <string>Enlever le diaporama</string>
   </property>
  </action>
  <action name="actionCharger_un_diaporama">
   <property name="text">
    <string>Charger un diaporama</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
